README.md

```diff
@@ -1,15 +1,17 @@
-[![Build Status](https://cloud.drone.io/api/badges/packethost/hegel/status.svg)](https://cloud.drone.io/packethost/hegel)
+[![Build Status](https://cloud.drone.io/api/badges/tinkerbell/hegel/status.svg)](https://cloud.drone.io/tinkerbell/hegel)
 
 ### Hegel
-The logical successor to Kant? A gRPC metadata service for Packet. Subscribe to changes in device metadata, get notified when IPs are added/removed, a device appears in the project, spot instance termination is triggered, etc.
+The logical successor to Kant? The gRPC and HTTP metadata service for Tinkerbell. Subscribe to changes in metadata, get notified when data is added/removed, etc. Full documentation can be found at [tinkerbell.org](https://github.com/tinkerbell/tink)
```

Comment:
since you're already making typographical fixes can you split this to one line per sentence too?

Reply:
PTAL